.github/docker-soak-tests/otel-collector/collector-config.yml

```diff
@@ -0,0 +1,66 @@
+receivers:
+  otlp:
+    protocols:
+      grpc:
+        endpoint: 0.0.0.0:4317
+  hostmetrics:
+    collection_interval: ${HOSTMETRICS_COLLECTION_INTERVAL_IN_SECONDS}s
+    scrapers:
+      process:
+        include:
+          match_type: strict
+          names:
+            - ${PROCESS_EXECUTABLE_NAME}
+
+processors:
+  filter:
+    metrics:
+      include:
+        match_type: strict
+        metric_names:
+          - process.cpu.time
+          # - process.disk.io
+          - process.memory.physical_usage
+          - process.memory.virtual_usage
+        resource_attributes:
+          - Key: process.command_line
+            Value: ${PROCESS_COMMAND_LINE}
+
+exporters:
+  logging/debug:
+    loglevel: debug
+  logging/error:
```

Comment:
Don't think you need the error logging you could have the debug logging exporter, and whether it's on or not is controlled  by `--log-level` when executing

Reply:
That's an interesting call out, thanks! I guess `--log-level` is at the collector level and `loglevel` is for the `logging:` exporter specifically? But then `--log-level` trumps `loglevel`?

The logs were pretty noise-y as is so I'll test out having `--log-leve=error` and just a simple

```yaml
exporters:
  logging:
```

Reply:
So I tried using `--log-level error` in docker-compose.yml and 

```yaml
exporters:
  logging:
```

in `collector-config.yml` but I still see debug logs:

```
{"#spans": 512}
otel_1            | 2021-09-08T18:01:50.346Z	INFO	loggingexporter/logging_exporter.go:41	TracesExporter	{"#spans": 512}
otel_1            | 2021-09-08T18:01:51.730Z	INFO	loggingexporter/logging_exporter.go:41	TracesExporter	{"#spans": 512}
otel_1            | 2021-09-08T18:01:53.122Z	INFO	loggingexporter/logging_exporter.go:41	TracesExporter	{"#spans": 512}
```

I'll fiddle with these 2 values to see if I missed something, please lmk if you can see something I did wrong!